Sources/MapboxDirections/RouteLeg.swift

```diff
@@ -236,6 +267,10 @@ open class RouteLeg: Codable {
      Do not assume that the user would travel along the leg at a fixed speed. For the expected travel time on each individual segment along the leg, use the `RouteStep.expectedTravelTimes` property. For more granularity, specify the `AttributeOptions.expectedTravelTime` option and use the `expectedSegmentTravelTimes` property.
      */
     open var expectedTravelTime: TimeInterval
+
+    open var administrationRegions: [AdministrationRegion]?
+
+    open var incidents: [Incident]?
```

Comment:
This property is undocumented. We shouldn’t use it in the navigation SDK until we document it.